PluginInstaller/wix-template.xml

```diff
@@ -1,132 +1,90 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-    <?define ProductVersion="<?version-long>" ?>
-    <?define UpgradeCode="8282b90a-cb51-4c02-a1c1-ecfcff9861bf"?>
-    <?define ProductCode="67d68e4a-82d2-4a7d-a909-cfce92dbe71a"?>
-    <?define VersionShort="<?version-short>" ?>
-    <Product Id="$(var.ProductCode)" Name="UserPlugins" Language="1033" Version="$(var.ProductVersion)" Manufacturer="Default Manufacturer" UpgradeCode="$(var.UpgradeCode)">
-        <Package Description="%PRODUCT_NAME% Plugins" InstallerVersion="200" Compressed="yes" />
-        <Media Id="1" Cabinet="UserPluginscab.cab" EmbedCab="yes" />
-        <Directory Id="TARGETDIR" Name="SourceDir">
-            <Directory Id="ProgramFilesFolder">
-                <Directory Id="%MANUFACTURER_NAME%" Name="%MANUFACTURER_NAME%">
-                    <Directory Id="INSTALLDIR" Name="%HYPERVISOR_NAME%">
-                        <Directory Id="XCPlugins" Name="Plugins" />
-                    </Directory>
-                </Directory>
-            </Directory>
-            <Directory Id="ProgramMenuFolder" />
-            <Directory Id="DesktopFolder" />
-        </Directory>
-
-  <UI Id="WixUI_Mondo">
-    <!--
-    <TextStyle Id="WixUI_Font_Normal" FaceName="Tahoma" Size="8" />
-    <TextStyle Id="WixUI_Font_Bigger" FaceName="Tahoma" Size="12" />
-    <TextStyle Id="WixUI_Font_Title" FaceName="Tahoma" Size="9" Bold="yes" />
-
-    <Property Id="DefaultUIFont" Value="WixUI_Font_Normal" />
-
-    <Property Id="WixUI_Mode" Value="Minimal" />
-    -->
-
-    <DialogRef Id="ErrorDlg" />
-    <DialogRef Id="FatalError" />
-    <DialogRef Id="FilesInUse" />
-    <DialogRef Id="MsiRMFilesInUse" />
-    <DialogRef Id="PrepareDlg" />
-    <DialogRef Id="ProgressDlg" />
-    <DialogRef Id="ResumeDlg" />
-    <DialogRef Id="UserExit" />
-    <DialogRef Id="InstallDirDlg" />
+  <?define PLUGIN_VERSION="%PLUGIN_VERSION%" ?>
```

Comment:
Renaming UpgradeCode to UPGRADE_CODE makes it more grepable. And since in the new code there will add 6 more new parameters, staying the old naming will make them all hard to grep.